llm-service/app/tests/conftest.py

```diff
@@ -55,13 +55,19 @@
 from moto import mock_aws
 
 from app.ai.vector_stores.qdrant import QdrantVectorStore
+from app.config import settings
 from app.main import app
 from app.services import models, data_sources_metadata_api
 from app.services.data_sources_metadata_api import RagDataSource
 from app.services.noop_models import DummyLlm
 from app.services.utils import get_last_segment
 
 
+# This is a hook to configure the pytest session
+def pytest_configure(config: Any) -> None:
+    settings.rag_databases_dir = "/tmp/databases"
```

Comment:
Any particular reason why we're doing this hard-coded path instead of something like https://docs.pytest.org/en/stable/how-to/tmp_path.html?

Reply:
pure python ignorance is all